packages/fuselage-toastbar/src/ToastBar.spec.tsx

```diff
@@ -1,12 +1,63 @@
 import { composeStories } from '@storybook/testing-react';
-import { render } from '@testing-library/react';
+import { render, getByRole, screen, fireEvent } from '@testing-library/react';
```

Comment:
`question` Why using `fireEvent` instead of `userEvent`?

Reply:
Yes, I think we can take advantage whenever comes to catching undesired side-effects, as a user interaction triggers many events.

Reply:
alright then, replacing it!